src/main/java/com/rewedigital/composer/ComposerApplication.java

```diff
@@ -20,33 +20,47 @@
 
 public class ComposerApplication {
 
-    public static void main(final String[] args) throws LoadingException {
+    static final String COMPOSER = "composer";
 
-        final Service service =
-            HttpService
-                .usingAppInit(Initializer::init, "composer")
-                .withModule(HttpClientModule.create())
-                .withModule(ClientDecoratingModule.create(new ErrorClientDecorator()))
-                .build();
+    public static void main(final String[] args) throws LoadingException {
+        HttpService.boot(bootstrapService(), args);
+    }
 
-        HttpService.boot(service, args);
+    private static Service bootstrapService() {
+        return HttpService
+            .usingAppInit(Initializer::init, COMPOSER)
+            .withEnvVarPrefix(COMPOSER.toUpperCase())
+            .withModule(HttpClientModule.create())
+            .withModule(ClientDecoratingModule.create(new ErrorClientDecorator()))
+            .build();
     }
 
-    private static class Initializer {
+    static class Initializer {
 
-        private static void init(final Environment environment) {
+        static void init(final Environment environment) {
             final Config configuration = withDefaults(environment.config());
 
             final ComposingRequestHandler handler =
                 new ComposingRequestHandler(
-                    new BackendRouting(configuration.getConfig("composer.routing")),
+                    new BackendRouting(configuration.getConfig(COMPOSER + ".routing")),
```

Comment:
I'm not sure if the constant `COMPOSER` improves readability...

Reply:
Agree..